Inspicio/Controllers/ImagesController.cs

```diff
@@ -249,132 +282,63 @@ public class DataFromBody
             }
 
-            // if the review has been rejected
-            if (data.ThumbChosen == (int)Status.REJECTED)
+            if ( data.state == States.NeedsWork )
             {
-                if (review.Rejected == false)
-                {
-                    if (review.NeedsWork == true)
-                    {
-                        review.Rejected = true;
-                        review.NeedsWork = false;
-                        image.NoOfRejections++;
-                        if (image.NoOfNeedsWork > 0)
-                        {
-                            image.NoOfNeedsWork--;
-                        }
-                    }
-                    else if (review.Approved == true)
-                    {
-                        review.Approved = false;
-                        review.Rejected = true;
-                        image.NoOfRejections++;
-                        if (image.NoOfApprovals > 0)
-                        {
-                            image.NoOfApprovals--;
-                        }
-                    }
-                    else if ((review.Approved == false) && (review.NeedsWork == false) && (review.Rejected == false))
-                    {
-                        review.Rejected = true;
-                        image.NoOfRejections++;
-                    }
-                }
+                 if (review.State != Review.States.NeedsWork)
+                 {
+                    review.State = Review.States.NeedsWork;
+                 }
             }
 
-            // if the review needs more work
-            if (data.ThumbChosen == (int)Status.NEEDSWORK)
+            if (data.state == States.Rejected)
             {
-                if (review.NeedsWork == false)
+
+                if (review.State != Review.States.Rejected)
                 {
-                    if (review.Approved == true)
-                    {
-                        review.NeedsWork = true;
-                        review.Approved = false;
-                        image.NoOfNeedsWork++;
-                        if (image.NoOfApprovals > 0)
-                        {
-                            image.NoOfApprovals--;
-                        }
-                    }
-                    else if (review.Rejected == true)
-                    {
-                        review.Rejected = false;
-                        review.NeedsWork = true;
-                        image.NoOfNeedsWork++;
-                        if (image.NoOfRejections > 0)
-                        {
-                            image.NoOfRejections--;
-                        }
-                    }
-                    else if ((review.Approved == false) && (review.NeedsWork == false) && (review.Rejected == false))
-                    {
-                        review.NeedsWork = true;
-                        image.NoOfNeedsWork++;
-                    }
+                    review.State = Review.States.Rejected;
                 }
```

Comment:
Also, it doesn't matter if it's already set to the same value, so you could simplify a bit more. e.g. instead of 
```
            if (data.state == States.Approved)
            {
                if (review.State != Review.States.Approved)
                {
                    review.State = Review.States.Approved;
                }
            }

            if ( data.state == States.NeedsWork )
            {
                 if (review.State != Review.States.NeedsWork)
                 {
                    review.State = Review.States.NeedsWork;
                 }
            }

            if (data.state == States.Rejected)
            {

                if (review.State != Review.States.Rejected)
                {
                    review.State = Review.States.Rejected;
                }
            }  
```

you should be able to just use
```
            if (data.state == States.Approved)
            {
                review.State = Review.States.Approved;
            }
            else if ( data.state == States.NeedsWork )
            {
                review.State = Review.States.NeedsWork;
            }
            else if (data.state == States.Rejected)
            {
                review.State = Review.States.Rejected;
            }
```

Or, if you used the same enum type in both places, you should be able to just use

 `review.State = data.state;` 

without any need for for the if statements.